.travis/rpmbuild_test

```diff
@@ -1,12 +1,13 @@
 #!/bin/bash -ex
 
-# this script runs as root under docker 
+# this script runs as root under docker
 
 OS_VERSION="$1"
 
 # build and install
-yum install -y libtool rpm-build make squashfs-tools openssl-devel libuuid-devel
+yum install -y libtool rpm-build make yum-utils squashfs-tools
```

Comment:
Remind me, did we need squashfs-tools here for the build to complete?  If not, I think we should remove it.

Reply:
nvrmnd.  I see that we do need it.  Sorry for the noise.

Reply:
+1, I've hit this bug a bunch of times forgetting to install it :)

Reply:
Yeah...

>GodloveD commented 8 days ago
Hmm, yeah I noticed that. squashfs-tools does need to be present on the host afaik. Maybe adding it back in will lead to a passing build?

>dctrud Add back squashfs-tools to rpmbuild_test script Verified 6ba0a1b

It was needed previously, had to add back in to make the build complete and get a travis checkmark. But... maybe this has been fixed by other things from the merges into development since then? Can't check right now if removing it is okay - but can tonight if needed.

Reply:
Ahh okay sounds like its needed  :-)

Reply:
Things actually may have changed because @jmstover did some work recently to clean up dependencies in our test suite.  But don't worry I'm having a look myself.  Will let you know.  Thanks!